internal/sdk/cloudian/secret_test.go

```diff
@@ -0,0 +1,24 @@
+package cloudian
+
+import (
+	"encoding/json"
+	"testing"
+)
+
+func TestSecretUnmarshal(t *testing.T) {
+	jsonString := `[{"accessKey":"124","secretKey":"x+2","createDate":1735894172440,"active":true}]`
```

Comment:
Don't think we need to test the builtin go marshalling. Suggesting to simply remove the test.

Obfuscating also seems a bit overkill, as the thing running this has access, and we should not print the thing in production.